msi/GitProduct.wxs

```diff
@@ -58,11 +62,25 @@
     <SetProperty Id="ENABLEFSCACHE" Value="[CMDLINE_ENABLEFSCACHE]" After="AppSearch" Sequence="first">CMDLINE_ENABLEFSCACHE</SetProperty>
 
     <!-- ENABLEGCM Property implementing the "Remember Property" pattern -->
-    <Property Id="ENABLEGCM" Value="1">
+    <!-- Initially set this to disabled until we can determine that .NET Framework 4.5.1 and above is installed. -->
+    <Property Id="ENABLEGCM" Value="0" />
```

Comment:
I still think that the logic could be simplified. How about the following:

- if the user sets `ENABLEGCM:(0|1)` on the command-line, it wins (otherwise it will remain on the previous `-1`)
- otherwise, if we find the registry setting, it wins (otherwise it will remain on the initial `-1`)
- otherwise detect .NET >= 4.5.1, if it exists, install and enable GCM, otherwise do not.

Always record in the registry whether GCM was enabled.

How does that sound?